docs/_docs/templates.md

```diff
@@ -416,9 +416,14 @@ The default is `default`. They are as follows (with what they filter):
 - `default`: spaces and non-alphanumeric characters
 - `pretty`: spaces and non-alphanumeric characters except for `._~!$&'()+,;=@`
 
-## Tags
+## Jekyll Tags
```

Comment:
changing this heading alone will break any url bookmarked to former `/templates/#tags`